docs/trace/data-prepper.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Data Prepper
+parent: Trace Analytics
+nav_order: 20
+---
+
+# Data Prepper
+
+Data Prepper is an independent component, not an Elasticsearch plugin, that converts data for use with Elasticsearch. It's not bundled with the all-in-one Open Distro for Elasticsearch installation packages.
+
+
+## Install Data Prepper
+
+To use the Docker image, pull it like any other image:
+
+```bash
+docker pull <>
+```
+
+Otherwise, [download](https://opendistro.github.io/for-elasticsearch/downloads.html) the appropriate TAR file for your operating system and unzip it.
+
+
+## Configure pipelines
+
+To use Data Prepper, you define pipelines in a configuration YAML file. Each pipeline is a combination of a source, a buffer, zero or more processors, and one or more sinks:
```

Comment:
buffer is optional - if the configuration file does not have buffer defined, we will default to `bounded_blocking` with default values

Reply:
I'm OK with the wording here, because even if people don't specify a buffer, every pipeline has one. Like you said, we cover the "you can omit this" case shortly after introducing the term.